.bumpversion_client.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.0.0-a0
+current_version = 2.0.0a1
```

Comment:
is that one intentional?

Reply:
Yeah, we haven't been using bumpversion recently... 
It doesn't really add much value anymore since the only version in a file we have left is the one in `docs/conf.py` and the tag is generated by github.